src/assets/css/bootstrap.min.css

```diff
@@ -6023,6 +6023,7 @@ fieldset:disabled .btn {
 }
 .modal-body {
   position: relative;
+  margin: auto;
```

Comment:
I'd highly suggest to not modify libraries source code (I mean, even though we have it locally, imagine that in a future, someone that is not aware of the custom css in this file decides to upgrade the version to something newer. The custom css will be gone). What can I do? Create a custom css file and override the bootstrap styles. That way we'll make sure our own rules wi'll stay there no matter what changes we do to the main library file.